src/OmnichannelChatSDK.ts

```diff
@@ -327,6 +332,15 @@ class OmnichannelChatSDK {
         });
 
         const sessionCloseOptionalParams: ISessionCloseOptionalParams = {};
+ 
+        if (this.isPersistentChat && !this.chatSDKConfig.persistentChat?.disable) {
+            if (this.reconnectId !== null) {
+                this.isReconnectChat = true;
```

Comment:
You don't need `this.isReconnectChat` since it's being used only once. You can write it as follow

```
const isReconnectChat = this.reconnectId !== null? true: false;
```